lib/include/ert/util/vector_util.hpp

```diff
@@ -0,0 +1,59 @@
+/*
+   Copyright (C) 2018  Statoil ASA, Norway.
+
+   The file 'vector_util.h' is part of ERT - Ensemble based Reservoir Tool.
+
+   ERT is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or
+   FITNESS FOR A PARTICULAR PURPOSE.
+
+   See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+   for more details.
+*/
+
+#include <stdio.h>
+
+#include <vector>
+
+template <class T>
+int vector_util_index(const std::vector<T>& vec, T value) {
```

Comment:
A bit skeptical to add this - should be possible to implement equivalent functionality with `std::find`
```
auto iter = std::find(vec.begin(), vec.end, value);
if (iter == vec.end())
    index = -1;
else
    index = iter - vec.begin();
```

Reply:
It is refactoring . This is a replacement of a function from @type@_vector_index. If the function is not replaced, we will have to use 5 lines of code every time we want to find an index to a value.